### ISSUE TITLE
Installing a beta version from an official repository

### ISSUE THREAD

**Issue by mpeter50** [doublecmd/doublecmd#375]
I was trying to install Double Commander to openSUSE Leap 15.3 from [this](https://build.opensuse.org/project/repository_state/home:Alexx2000:doublecmd-svn/openSUSE_Leap_15.3) repository.
The problem is, I wasn't able to run it because of an immediate segfault, and after a little investigation I've found that I got the 1.1.0 (qt5) version.

It was my understanding that the repositories serve the latest beta version, which is 1.0.4 as of now.
Is it possible to have Double Commander automatically updated to the beta versions through a repository?

Also, an other question is, where are the sources of the 1.1.0 version? I wasn't able to find such a tag or branch in the repo.

**Comment by JonnyTech** [comment]
Use the repository at https://download.opensuse.org/repositories/home:/Alexx2000/ rather than https://download.opensuse.org/repositories/home:/Alexx2000:/doublecmd-svn/ for the beta releases.

**Comment by mpeter50** [comment]
Thanks, I'll try it.

Do you know where can I find this link for future reference? I have only seen the second one in the Wiki.

**Comment by alexx2000** [comment]
Also see https://doublecmd.github.io/doc/en/faq.html#repo

**Comment by mpeter50** [comment]
Hmm, I've changed the repository to `https://download.opensuse.org/repositories/home:/Alexx2000/openSUSE_Leap_15.3/`, refreshed the cache, reinstalled `doublecmd-qt` and it still segfaults.

What am I doing wrong?
![image](https://user-images.githubusercontent.com/83356418/158709523-8b55b557-74eb-4f96-8396-b155815cc0bc.png)

**Comment by JonnyTech** [comment]
Try removing the doublecmd.cfg file, or run from a different user account.

**Comment by j2969719** [comment]
`coredumpctl info doublecmd`

**Comment by mpeter50** [comment]
> Try removing the doublecmd.cfg file, or run from a different user account.

I haven't found such a file, and according to `sudo find / -name doublecmd.cfg` there is no such file on my system.

> `coredumpctl info doublecmd`

```
           PID: 2890 (doublecmd)
           UID: 1000 (peter)
           GID: 100 (users)
        Signal: 11 (SEGV)
     Timestamp: Thu 2022-03-17 11:30:08 CET (4min 12s ago)
  Command Line: doublecmd
    Executable: /usr/lib64/doublecmd/doublecmd
 Control Group: /user.slice/user-1000.slice/user@1000.service/app.slice/app-org.kde.konsole-fae8c46699a9498eb546185701d549bf.scope
          Unit: user@1000.service
     User Unit: app-org.kde.konsole-fae8c46699a9498eb546185701d549bf.scope
         Slice: user-1000.slice
     Owner UID: 1000 (peter)
       Boot ID: e03de4d0f77a4cbeaa3fb09f737729ed
    Machine ID: 6f591f22252a481a9740787ac8ccf8ae
       Storage: /var/lib/systemd/coredump/core.doublecmd.1000.e03de4d0f77a4cbeaa3fb09f737729ed.2890.1647513008000000.zst
       Message: Process 2890 (doublecmd) of user 1000 dumped core.
                
                Stack trace of thread 2890:
                #0  0x00007fab92b99ed8 _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dced8)
                #1  0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #2  0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #3  0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #4  0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #5  0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #6  0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #7  0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #8  0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #9  0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #10 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #11 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #12 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #13 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #14 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #15 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #16 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #17 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #18 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #19 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #20 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #21 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #22 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #23 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #24 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #25 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #26 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #27 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #28 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #29 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #30 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #31 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #32 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #33 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #34 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #35 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #36 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #37 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #38 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #39 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #40 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #41 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #42 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #43 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #44 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #45 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #46 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #47 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #48 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #49 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #50 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #51 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #52 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #53 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #54 0x00007fab92b99edd _ZNK6QColor6darkerEi (libQt5Pas.so.1 + 0x1dcedd)
                #55 0x00007fab905b922d _ZNK6QColor4darkEi (libQt5Gui.so.5 + 0x35a22d)
                #56 0x00007fab92b99edd _ZNK6QCol
```

**Comment by j2969719** [comment]
you can try to redirect the output to a file and attach it here, it looks like some stack trace lines are missing

**Comment by mpeter50** [comment]
It might be interesting that I've just installed media codecs with `opi codecs`, and it downgraded the libQt5Pas1 package to version `2.2.0-pm153.5.1`.
As a result, DCMD now works (it opens and lists the content of a directory on both sides, but haven't tried to do anything in it).

From the stable DCMD repository I got version `2.6-lp153.8.1` of libQt5Pas1, though.

For now I switched back to that so I can help.
